warehouse/models/rt_staging/_rt_staging.yml

```diff
@@ -0,0 +1,55 @@
+version: 2
+
+sources:
+  - name: gtfs_rt_external_tables
+    description: New hive-partitioned external tables reading from GCS.
```

Comment:
Don't think we need to call these "new" any more do we?

Reply:
nah, will change